runtime-modules/working-group/src/tests/fixtures.rs

```diff
@@ -661,17 +661,15 @@ impl TerminateWorkerRoleFixture {
         let actual_result = TestWorkingGroup::terminate_role(
             self.origin.clone().into(),
             self.worker_id,
-            self.penalty.clone(),
+            self.penalty,
             self.rationale.clone(),
         );
         assert_eq!(actual_result, expected_result);
 
-        if actual_result.is_ok() {
-            if actual_result.is_ok() {
-                assert!(!<crate::WorkerById<Test, DefaultInstance>>::contains_key(
-                    self.worker_id
-                ));
-            }
+        if actual_result.is_ok() && actual_result.is_ok() {
```

Comment:
```suggestion
        if actual_result.is_ok() {
```

Reply:
done in eb7e7556253695db2dd22ec1d437974758e26976